lib/Document.php

```diff
@@ -160,7 +160,7 @@ public function download(?bool $postProcess = true, ?string $encoding = null): s
             case ContentType::XLSX:
                 $spreadsheet = $reader->load($this->tmpFilename);
                 if ($this->contentType !== ContentType::XLSX) {
-                    $sheet = $spreadsheet->getSheet(0)->toArray();
+                    $sheet = $spreadsheet->getSheet(0)->toArray(null, false);
```

Comment:
would you mind just linking to the docs for `toArray` so i understand what's going on here? i just did a quick search of their docs and didn't see any `toArray` parameter documentation

Reply:
You can check toArray parameters at https://phpoffice.github.io/PhpSpreadsheet/classes/PhpOffice-PhpSpreadsheet-Worksheet-Worksheet.html#method_toArray

Reply:
thank you